### DISCUSSION TITLE
Write metadata to .wav file?

### DISCUSSION THREAD

**Discussion by rasmushauschild** [pschatzmann/arduino-audio-tools#457] [Q&A] [answered]
Hi,
I'm wondering if it's possible to write metadata like album title (IPRD), text comments (ICMT), etc. to a .wav file created with the included `WAVEEncoder()`?

Thanks in advance!
Rasmus

**Comment by pschatzmann** [comment, marked as answer]
No, I was't even aware that wav files can support metadata. 
Feel free to extend the logic. The current logic is pretty simple to just prefix the pcm stream with a fixed header, so that external applications can process the data.

**Reply by rasmushauschild** [reply]
Thanks for the clarification! I'll look into it.
